Security/src/CloudFoundrySingleSignon/README-UAA.md

```diff
@@ -0,0 +1,122 @@
+
+1. uaac target uaa.`YOUR-CLOUDFOUNDRY-SYSTEM-DOMAIN` (e.g. `uaac target uaa.system.testcloud.com`)
+
+Next, authenticate and obtain an access token for the `admin client` from the UAA server so that we can add our new application/user credentials. You will need the `Admin Client Secret` for your installation of CloudFoundry for this step. If you are using Tanzu Application Service (TAS), you can obtain the secret from the `Ops Manager/Elastic Runtime` credentials page under the `UAA` section.  Look for `Admin Client Credentials` and then use it as follows:
+
+1. uaac token client get admin -s `ADMIN_CLIENT_SECRET`
+
+> Note: To see the token that was retrieved, run the command `uaac contexts`
+
+### Add User and Group
+
+After authenticating, add a new `user` and `group` to the UAA Server database. Do NOT change the group name: `testgroup` as it is used for policy based authorization in the sample application. Feel free to change the username and password to anything you would like.
+
+1. uaac group add testgroup
+1. uaac user add dave --given_name Dave --family_name Tillman --emails dave@testcloud.com --password Password1!
+1. uaac member add testgroup dave
+
+### Add New Client for our App
+
+After adding the user and group, we are ready to add our application as a new client to the UAA server. This step will establish our application's credentials and allow it to interact with the UAA server. Use the line below once you have replaced the `YOUR-CLOUDFOUNDRY-APP-DOMAIN` with the domain used by your cloud foundry instance.
+
+```bash
+uaac client add myTestApp --scope cloud_controller.read,cloud_controller_service_permissions.read,openid,testgroup --authorized_grant_types authorization_code,refresh_token --authorities uaa.resource --redirect_uri http://single-signon.`YOUR-CLOUDFOUNDRY-APP-DOMAIN`/signin-cloudfoundry --autoapprove cloud_controller.read,cloud_controller_service_permissions.read,openid,testgroup --secret myTestApp
+```
+
+### Add User-Provided Service with OAuth Details
+
+Last, we will create a user-provide service that includes the appropriate UAA server configuration data. Use the sample below to pass the parameters directly to the `cf cups` command, replacing `<YOUR-CLOUDFOUNDRY-SYSTEM-DOMAIN>` with your domain.
+
+```bash
+cf target -o myorg -s development
+cf cups myOAuthService -p "{\"client_id\": \"myTestApp\",\"client_secret\": \"myTestApp\",\"uri\": \"uaa://login.<YOUR-CLOUDFOUNDRY-SYSTEM-DOMAIN>\"}"
+```
+
+## Publish App & Push to CloudFoundry
+
+1. cf target -o myorg -s development
+1. cd samples/Security/src/CloudFoundrySingleSignon
+1. dotnet restore --configfile nuget.config
+1. Publish app to a directory selecting the framework and runtime you want to run on.
+    * `dotnet publish -f netcoreapp3.1 -r linux-x64`
+    * `dotnet publish -f netcoreapp3.1 -r win10-x64`
```

Comment:
We use 'win-x64' in the connector samples instead of 'win10-x64', which is a bit more maintainable.

```suggestion
    * `dotnet publish -f netcoreapp3.1 -r win-x64`
```